src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.h

```diff
@@ -25,148 +25,165 @@
 class vtkPlusDataCollectionExport vtkPlusWinProbeVideoSource : public vtkPlusUsDevice
 {
 public:
-    /*! Constructor for a smart pointer of this class*/
-    static vtkPlusWinProbeVideoSource * New();
-    vtkTypeMacro(vtkPlusWinProbeVideoSource, vtkPlusDevice);
-    virtual void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;
+  /*! Constructor for a smart pointer of this class*/
+  static vtkPlusWinProbeVideoSource* New();
+  vtkTypeMacro(vtkPlusWinProbeVideoSource, vtkPlusDevice);
+  virtual void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;
 
-    /*! Specify the device connected to this class */
-    virtual bool IsTracker() const { return false; }
+  /*! Specify the device connected to this class */
+  virtual bool IsTracker() const { return false; }
 
-    /*! Read configuration from xml data */
-    virtual PlusStatus ReadConfiguration(vtkXMLDataElement* config);
+  /*! Read configuration from xml data */
+  virtual PlusStatus ReadConfiguration(vtkXMLDataElement* config);
 
-    /*! Write configuration to xml data */
-    virtual PlusStatus WriteConfiguration(vtkXMLDataElement* config);
+  /*! Write configuration to xml data */
+  virtual PlusStatus WriteConfiguration(vtkXMLDataElement* config);
 
-    /* Set the transmit frequency of US probe (MHz) */
-    PlusStatus SetTxTxFrequency(float frequency);
+  /* Set the transmit frequency of US probe (MHz) */
+  PlusStatus SetTxTxFrequency(float frequency);
 
-    /* Get the transmit frequency of US probe (MHz) */
-    float GetTxTxFrequency();
+  /* Get the transmit frequency of US probe (MHz) */
+  float GetTxTxFrequency();
 
-    /* Set the voltage of US probe (percent?) */
-    PlusStatus SetVoltage(uint8_t voltage);
+  /* Set the voltage of US probe (percent?) */
+  PlusStatus SetVoltage(uint8_t voltage);
 
-    /* Get the voltage of US probe (percent?) */
-    uint8_t GetVoltage();
+  /* Get the voltage of US probe (percent?) */
+  uint8_t GetVoltage();
 
-    /* Set the scan depth of US probe (mm) */
-    PlusStatus SetSSDepth(float depth);
+  /* Set the scan depth of US probe (mm) */
+  PlusStatus SetSSDepth(float depth);
 
-    /* Get the scan depth of US probe (mm) */
-    float GetSSDepth();
+  /* Get the scan depth of US probe (mm) */
+  float GetSSDepth();
```

Comment:
No need to save characters, easier for autocomplete tools if the user could type "scan" and this method would show up.

Recommend GetScanDepth unless you think otherwise.

Reply:
And as per @lassoan suggestion, add unit to end of function name

Reply:
OK I will systematically review and change method and ivar names.